workspaces/global-header/plugins/global-header/src/components/GlobalHeader.tsx

```diff
@@ -0,0 +1,377 @@
+import ManageAccountsOutlinedIcon from '@mui/icons-material/ManageAccountsOutlined';
+import LogoutOutlinedIcon from '@mui/icons-material/LogoutOutlined';
+import { Link } from '@backstage/core-components';
+import SearchIcon from '@mui/icons-material/Search';
+import Box from '@mui/material/Box';
+import AccountCircleOutlinedIcon from '@mui/icons-material/AccountCircleOutlined';
+
+const models = [
+  {
+    key: 1,
+    value: 'argocd-template',
+    label: 'Add ArgoCD to an existing project',
+  },
+  {
+    key: 2,
+    value: 'create-backend-plugin',
+    label: 'Create Backend Plugin Template',
+  },
+  {
+    key: 3,
+    value: 'create-frontend-plugin',
+    label: 'Create Frontend Plugin Template',
+  },
+  {
+    key: 4,
+    value: 'create-react-app-template-test-annotator',
+    label: 'Create React App Template with annotations',
+  },
+  {
+    key: 5,
+    value: 'define-ansible-job-template',
+    label: 'Ansible Job Template',
+  },
+];
+
+interface MenuSectionProps {
+  children: React.ReactNode;
+  label: string;
+}
+
+const MenuSectionRoot = styled('li')`
+  list-style: none;
+
+  & > ul {
+    padding-left: 0;
+  }
+`;
+
+const MenuSectionLabel = styled('span')`
+  display: inline-block;
+  font-size: 0.875em;
+  alignment-baseline: baseline;
+  align-content: end;
+  color: #979797;
+`;
+
+function MenuSection({ children, label }: Readonly<MenuSectionProps>) {
+  return (
+    <MenuSectionRoot role="group">
+      <Box
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          margin: '0.75rem',
+        }}
+      >
+        <MenuSectionLabel>{label}</MenuSectionLabel>
+        <MenuItem>
+          <Link
+            underline="none"
+            to="/create"
+            style={{ display: 'inline-block', fontSize: '0.875em' }}
+          >
+            All templates
+          </Link>
+        </MenuItem>
+      </Box>
+      <ul style={{ marginLeft: 12, marginRight: 12 }}>{children}</ul>
+    </MenuSectionRoot>
+  );
+}
+
+export const GlobalHeader = () => {
+  const Search = styled('div')(({ theme }) => ({
+    position: 'relative',
+    borderRadius: `${theme.shape.borderRadius}`,
+    backgroundColor: 'transparent',
+    marginRight: theme.spacing(2),
+    marginLeft: 0,
+    width: '100%',
+    [theme.breakpoints.up('sm')]: {
+      marginLeft: 0,
+      width: 'auto',
+    },
+  }));
+
+  const SearchIconWrapper = styled('div')(({ theme }) => ({
+    padding: theme.spacing(0, 2),
+    height: '100%',
+    position: 'absolute',
+    pointerEvents: 'none',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  }));
+
+  const StyledInputBase = styled(InputBase)({
+    color: 'inherit',
+    width: '100%',
+    '& input': {
+      paddingLeft: '3rem',
+    },
+  });
+
+  const Listbox = styled('ul')(
+    ({ theme }) => `
+    font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
We <3 IBM, but we don't want hard code they font here. ;)

Reply:
Oops, I copied this code from the Mui example. Good catch 👍